pkg/protostore/protostore.go

```diff
@@ -76,22 +77,34 @@ type Metadata struct {
 }
 
 type ProtoStoreMsg struct {
-	Id         string `gorm:"primaryKey"`
-	Msg        []byte
-	ParentId   string
-	Revision   int64
-	OrgId      string `gorm:"primaryKey"`
-	InstanceId string `gorm:"primaryKey"`
-	CreatedAt  time.Time
-	UpdatedAt  time.Time
-	DeletedAt  gorm.DeletedAt
-	XTableName string `gorm:"-"`
+	Id         string         `gorm:"primaryKey" json:"id"`
+	Msg        []byte         `json:"-"`
+	ParentId   string         `json:"parent_id,omitempty"`
+	Revision   int64          `json:"revision"`
+	OrgId      string         `gorm:"primaryKey" json:"org_id"`
+	InstanceId string         `gorm:"primaryKey" json:"instance_id,omitempty"`
+	CreatedAt  time.Time      `json:"-"`
+	UpdatedAt  time.Time      `json:"-"`
+	DeletedAt  gorm.DeletedAt `json:"-"`
+	XTableName string         `gorm:"-" json:"x_table_name"`
```

Comment:
If I do `json:"table_name"`, lint complains. So, had to to `json:"x_table_name"`

Reply:
do you need this?

Reply:
It might be useful in the logs... Not sure yet. I'll need to look at the logs of a consumer of _multi-tenant-persistence-for-saas_. I'll remove it if I think it's unnecessary.

Reply:
ok for now, might be better to avoid logging tablename here completely later.